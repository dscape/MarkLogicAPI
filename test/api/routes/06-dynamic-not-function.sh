#!/bin/sh
curl -vs localhost:6984/server/idonotexistyet/1 2>&1 \
 | sed -e 's/^\*.*//' \
 | grep '[<|{].[A-C|E-Z|a-z]'