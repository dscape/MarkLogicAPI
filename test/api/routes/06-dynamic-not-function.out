< HTTP/1.1 404 Not Found
< Content-type: text/plain; charset=UTF-8
< Server: MarkLogic
< Content-Length: 54
< Connection: close
{"error":"dxc_internal","reason":"Undefined function"}
